packages/domains-table/src/domains-table/style.scss

```
@@ -137,12 +142,44 @@
 		}
 	}
 
+	.domains-table-row__ssl-cell .domains-table-row__ssl-status-button {
```

```suggestion
	.domains-table-row__ssl-cell {
		gap: 4px;
	}

	.domains-table-row__ssl-cell .domains-table-row__ssl-status-button {
```
Aligns with the way we style other table cells (particularly the renewal status cell)